benchmarks/accelerate_opt/benchfile.py

```diff
@@ -1,66 +1,46 @@
-from milabench import gpu
 from milabench.pack import Package
 
-import subprocess
-import json, os
 
-# This is called to ensure the arch variable is set
-gpu.get_gpu_info()
-
-
-class TheBenchmark(Package):
-    if gpu.arch == "cuda":
-        requirements_file = "requirements.nvidia.txt"
-    elif gpu.arch == "rocm":
-        requirements_file = "requirements.amd.txt"
-    else:
-        raise ValueError(f"Unsupported arch: {gpu.arch}")
+class AccelerateBenchmark(Package):
+    base_requirements = "requirements.in"
 
     def make_env(self):
         env = super().make_env()
-        env["MILABENCH_CONFIG"] = json.dumps(self.config)
-        env["HF_HOME"] = self.config['dirs']['data']
```

Comment:
You need to keep HF_HOME here, otherwise the benchmark will write in $HOME.

Reply:
If not set, HF_HOME will use a subdirectory of XDG_CACHE_HOME, which Milabench sets here: https://github.com/mila-iqia/milabench/blob/master/milabench/pack.py#L319

It's possible setting XDG_CACHE_HOME is an overreach, we'll see.